app/src/main/java/com/jalasoft/routesapp/data/model/remote/CityRoute.kt

```diff
@@ -0,0 +1,11 @@
+package com.jalasoft.routesapp.data.model.remote
+
+data class CityRoute(
+    var Bus: List<Line> = listOf(),
```

Comment:
Maybe we have to consider that we can have other transportation methods besides bus or subway